odoo/addons/test_http/tests/test_static.py

```diff
@@ -249,6 +250,75 @@ def test_static14_download_not_found(self):
         self.assertEqual(res.status_code, 404)
 
 
+@tagged('post_install', '-at_install')
+class TestHttpStaticLogo(TestHttpStaticCommon):
+    @staticmethod
+    def img_data_to_web_data(img_base_64):
+        return image_process(img_base_64, size=(180, 0))
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        ResCompany = cls.env['res.company']
+        cls.default_logo_data = TestHttpStaticLogo.img_data_to_web_data(base64.b64decode(ResCompany._get_logo()))
```

Comment:
```suggestion
        cls.default_logo_data = cls.img_data_to_web_data(base64.b64decode(ResCompany._get_logo()))
```

Reply:
Indeed, thanks, it is shorter and more readable.